sklearn/inspection/_permutation_importance.py

```diff
@@ -41,11 +46,23 @@ def _calculate_permutation_scores(estimator, X, y, sample_weight, col_idx,
         feature_score = _weights_scorer(
             scorer, estimator, X_permuted, y, sample_weight
         )
-        scores[n_round] = feature_score
+        if isinstance(feature_score, dict):
+            for name, score in feature_score.items():
+                scores[name][n_round] = score
+        else:
+            scores[n_round] = feature_score
 
     return scores
 
 
+def _create_importances_bunch(baseline_score, score):
+    """Create output importances Bunch."""
```

Comment:
```suggestion
    """Compute the importances as the decrease in score.
    
    Parameters
    ----------
    baseline_score : ndarray of shape (n_features,)
        The baseline score without permutation.
    permuted_score : ndarray of shape (n_features, n_repeats)
        The permuted scores for the `n` repetitions.
            
    Returns
    -------
    importances : :class:`~sklearn.utils.Bunch`
        Dictionary-like object, with the following attributes.

        importances_mean : ndarray, shape (n_features, )
            Mean of feature importance over `n_repeats`.
        importances_std : ndarray, shape (n_features, )
            Standard deviation over `n_repeats`.
        importances : ndarray, shape (n_features, n_repeats)
            Raw permutation importance scores.
    """
```

Reply:
Let's force to pass some NumPy array and we can be explicit in the documentation.

Reply:
ah ok